packages/nuxeo-web-ui-ftest/features/step_definitions/support/fixtures/layouts.js

```diff
@@ -171,10 +171,26 @@ global.fieldRegistry.register(
 );
 global.fieldRegistry.register(
   'nuxeo-checkbox-aggregation',
-  (element) => element.element('paper-checkbox').getAttribute('aria-checked'),
+  (element) => {
+    if (element.getAttribute('collapsible')) {
+      element
+        .element('nuxeo-collapsible')
+        .element('paper-checkbox')
+        .getAttribute('aria-checked');
+    } else {
+      element.element('paper-checkbox').getAttribute('aria-checked');
+    }
```

Comment:
We are repeating a big portion of the code. What about? (_not tested!_)
```suggestion
    let el = element;
    if (element.getAttribute('collapsible')) {
      el = el.element('nuxeo-collapsible');
    }
    element.element('paper-checkbox').getAttribute('aria-checked');
```